data/awsrds/v0/config/tasks.json

```diff
@@ -0,0 +1,286 @@
+            "template",
+            "value"
+          ],
+          "title": "Criteria"
+        },
+        "from": {
+          "description": "The starting row to be selected, if both 'from' and 'to' equals 0 then all rows will be selected",
+          "instillAcceptFormats": [
+            "number"
+          ],
+          "instillShortDescription": "Starting of rows (check description for more details)",
+          "instillUIOrder": 2,
+          "instillUpstreamTypes": [
+            "reference",
+            "template",
+            "value"
+          ],
+          "title": "From",
+          "type":"number"
+        },
+        "to": {
+          "description": "The starting row to be selected, if both 'from' and 'to' equals 0 then all rows will be selected",
+          "instillAcceptFormats": [
+            "number"
+          ],
+          "instillShortDescription": "Ending of rows (check description for more details)",
+          "instillUIOrder": 3,
+          "instillUpstreamTypes": [
+            "reference",
+            "template",
+            "value"
+          ],
+          "title": "To",
+          "type":"number"
+        },
+        "table-name": {
+          "description": "The table name in the database to upsert data into",
+          "instillAcceptFormats": [
+            "string"
+          ],
+          "instillShortDescription": "Database Table Name",
+          "instillUIOrder": 0,
+          "instillUpstreamTypes": [
+            "reference",
+            "template",
+            "value"
+          ],
+          "title": "Table Name",
+          "type":"string"
+        }
+      },
+      "required": [
+        "table-name",
+        "criteria",
+        "from",
+        "to"
+      ],
+      "title": "Input",
+      "type": "object"
+    },
+    "output": {
+      "description": "Output",
+      "instillEditOnNodeFields": [
+        "json"
+      ],
+      "instillUIOrder": 0,
+      "properties": {
+        "rows": {
+          "description": "The rows returned from the query",
+          "instillEditOnNodeFields": [],
+          "instillUIOrder": 0,
+          "required": [],
+          "title": "Rows",
+          "type": "array",
+          "instillFormat": "array:semi-structured/json",
+          "items": {
+            "title": "Result",
+            "instillFormat": "semi-structured/json"
+          }
+        },
+        "status": {
+          "description": "Upsert status",
+          "instillFormat": "string",
+          "required": [],
+          "instillUIOrder": 0,
+          "title": "Status",
+          "type": "string"
+        }
+      },
+      "required": [
+        "status",
+        "rows"
+      ],
+      "title": "Output",
+      "type": "object"
+    }
+  },
+  "TASK_DELETE": {
+    "instillShortDescription": "Perform a delete operation to update existing records based on specified criteria",
+    "input": {
+      "instillUIOrder": 0,
+      "properties": {
+        "criteria": {
```

Comment:
@chuang8511 

The task will perform a delete operation for multiple data entries that fit the criteria (it could also be singular if the criteria are specific enough). I believe deleting singular data won't be a useful feature since it will be covered if the criteria specify a single object, especially if the criteria involve a unique column such as an ID. Also, if it's only one object, I do feel like it's better to do it on their own rather than using vdp. What do you think? Is there any specific cases? If not, I will proceed with specifying it.

Reply:
@zidanehakim 
Ah, I just want our document to clarify this delete is for multiple objects if it could do deletion for multiple objects.
I think deleting multiple objects is fine for my imagination of the use case.